setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="staking_deposit",
-    version='2.5.6',
+    version='2.7.0',
```

Comment:
This is missing the __version__ now. Not sure what's happening

Reply:
I've added `__version__` here.

Reply:
Restored `develop` commit there.